lib/compressIntegerAccessors.js

```diff
@@ -41,29 +45,40 @@ var unsignedComponentTypes = [
  * than necessary. The data is rewritten to the most optimal data type.
  *
  * @param {Object} gltf
+ * @param {Object} options
+ * @param {Array} options.semantics An array of semantics to compress
  */
-function compressIntegerAccessors(gltf) {
+function compressIntegerAccessors(gltf, options) {
+    var semantics = defaultValue(options.semantics, []);
+    var semanticsLength = semantics.length;
+    var promises = [];
+    for (var i = 0; i < semanticsLength; i++) {
+        var semantic = semantics[i];
+        promises.push(getAccessorsForSemantic(gltf, semantic, compressIntegerAccessor));
+    }
+    return Promise.all(promises).then(function() {
+        uninterLeaveAndPackBuffers(gltf);
+    });
+}
+
+function compressIntegerAccessor(gltf, primitive, accessorId, matchedSemantic) {
+    void(primitive, matchedSemantic);
```

Comment:
It keeps jsHint from failing. I'm explicitly stating that those arguments from the callback aren't used here.